### ISSUE TITLE
Mask R-CNN on TensorFlow

### ISSUE THREAD

**Issue by tony-2018** [mlperf/community#3]
It looks like ResNet50 is used in Caff2 but ResNet101 is used in TensorFlow.  I wonder if there is a requirement on which ResNet must be used.  Thanks, 

==
https://github.com/mlperf/training/tree/master/object_detection/caffe2
<training/object_detection/caffe2/>
Publication/Attribution
He, Kaiming, et al. "Mask r-cnn." Computer Vision (ICCV), 2017 IEEE International Conference on. IEEE, 2017.
We use a version of Mask R-CNN with a ResNet50 backbone.
==
https://github.com/mlperf/community/tree/master/object_detection/tensorflow
Mask R-CNN for Object Detection and Segmentation
This is an implementation of Mask R-CNN on Python 3, Keras, and TensorFlow. The model generates bounding boxes and segmentation masks for each instance of an object in the image. It's based on Feature Pyramid Network (FPN) and a ResNet101 backbone.
==
